### PR TITLE
fix(types): publish protocol types to npm

### PR DESCRIPTION
**Pull request by connorjclark**
see #5168

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -252,8 +252,9 @@ Releasing to npm consists of the following phases:
     1. On your local machine, pull from [upstream](https://github.com/GoogleChrome/puppeteer) and make sure the last commit is the one just merged.
     2. Run `git status` and make sure there are no untracked files.
         - **WHY**: this is to avoid adding unnecessary files to the npm package.
-    3. Run [`npx pkgfiles`](https://www.npmjs.com/package/pkgfiles) to make sure you don't publish anything unnecessary.
-    4. Run `npm publish`. This publishes the `puppeteer` package.
+    3. Run `yarn` to make sure the latest `lib/protocol.d.ts` is generated.
```
Comment (mathiasbynens) [inline CONTRIBUTING.md]: Is there an easy way to do this that does not require installing`yarn`?

Comment (connorjclark) [reply CONTRIBUTING.md]: This should work for `npm install` too. It's done in an install hook.

Could replace with `npm run install / yarn install`.

Comment (mathiasbynens) [inline CONTRIBUTING.md]: Let’s do that then — it’s more consistent with the other steps:

```suggestion
    3. Run `npm install` to make sure the latest `lib/protocol.d.ts` is generated.
```

**Review by mathiasbynens — APPROVED** [review]

**Comment by mathiasbynens** [comment]
Thanks!

**Pull request merged**